pom.xml

```diff
@@ -822,6 +822,30 @@
             </systemPropertyVariables>
             <argLine>${unitTestMemSize}</argLine>
           </configuration>
+          <executions>
+            <execution>
+              <id>reuseForks-false-tests</id>
+              <goals>
+                <goal>test</goal>
+              </goals>
+              <configuration>
+                <forkCount>${surefire.forkCount}</forkCount>
+                <reuseForks>false</reuseForks>
+                <includes>
+                  <include>**/TableIdTest.java</include>
+                  <include>**/HadoopCredentialProviderTest.java</include>
+                  <include>**/IdleRatioScanPrioritizerTest.java</include>
+                  <include>**/AssignmentWatcherTest.java</include>
+                  <include>**/TestCfCqSlice.java</include>
+                  <include>**/TestCfCqSliceSeekingFilter.java</include>
+                </includes>
+                <systemPropertyVariables>
+                  <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
+                </systemPropertyVariables>
+                <argLine>${unitTestMemSize}</argLine>
```

Comment:
This looks like it could work, but there's a few things:

1. I don't think it's necessary to include configuration in this specific execution that is already okay at the level above, directly under the plugin, because it *should* inherit it from there. So, the system properties, forkCount, and argLine can probably be omitted.
2. Declaring the second execution in the root pom will make it execute for every module, even if there's no files in that module that would match the pattern. This might be okay if it's not too costly. Otherwise, it might be worth placing this in each child pom that has matching files that would be affected.
3. Need to double check that the second execution doesn't clobber the reports generated by the first execution. Hopefully, it only adds new reports, and doesn't clobber, but I haven't checked.

Reply:
I made the changes for points 1 and 2. 

> Need to double check that the second execution doesn't clobber the reports generated by the first execution. Hopefully, it only adds new reports, and doesn't clobber, but I haven't checked.

I am looking into all the reporting now to be sure. Everything from the default execution seems to be normal.

Reply:
I see the second execution Id in the logs but I do not see it actively running any tests. Not quite sure what is going on.

Reply:
You can try troubleshooting by executing with debug mode in Maven. I believe the flag for that is `mvn -X`. Warning: it's quite spammy. Expect to do a lot of scrolling, or pipe the output to a file for inspection after it runs.

Reply:
The issue seems to stem from the fact it is inheriting the excluded tests from the parent configuration. So it excludes what I am trying to include. Not sure on the correct way to reset it at this moment.

Reply:
You could try adding `<excludes combine.self=”override” />` to the second execution to prevent the excludes from being inherited. I haven't tried this yet.

Reply:
This worked, I will go with it since it is shorter.